bin/build-release.sh

```diff
@@ -40,19 +42,25 @@ Usage: bin/build-release.sh [<storm.tar.gz>]
                               bin/build-release.sh.
 
   ENV
-    MIRROR        Specify Apache Storm Mirror to download from
-                    Default: ${MIRROR}
-    STORM_RELEASE       The targeted release version of Storm
-                    Default: ${STORM_RELEASE}
-    MESOS_RELEASE       The targeted release version of MESOS
-                    Default: ${MESOS_RELEASE}
+    MIRROR          Specify Apache Storm Mirror to download from
+                      Default: ${MIRROR}
+    STORM_RELEASE   The targeted release version of Storm
+                      Default: ${STORM_RELEASE}
+    MESOS_RELEASE   The targeted release version of MESOS
+                      Default: ${MESOS_RELEASE}
+    STORM_URL       Override the URL for downloading the storm binary
+                      NOTE: ensure version matches with STORM_RELEASE
 
 USAGE
 }; function --help { help ;}; function -h { help ;}
 
 function downloadStormRelease {
   if [ ! -f apache-storm-${STORM_RELEASE}.tar.gz ]; then
+    if [ -z ${STORM_URL} ]; then
       wget --progress=dot:mega ${MIRROR}/apache/storm/apache-storm-${STORM_RELEASE}/apache-storm-${STORM_RELEASE}.tar.gz
+    else
+      wget --progress=dot:mega ${STORM_URL}
+    fi
```

Comment:
Minor nit, use curl instead of wget.  Justification: I can easily use file:///local-storm-dist.tgz and avoid pushing the tgz to a webserver. Also, on more systems by default and supports more protocols (scp and sftp in particular).